ros2action/ros2action/verb/send_goal.py

```diff
@@ -79,12 +79,21 @@ def send_goal(action_name, action_type, goal_values, feedback_callback):
     node = None
     action_client = None
     try:
-        # TODO(jacobperron): This logic should come from a rosidl related package
-        package_name, action_type = action_type.split('/', 2)
-        if not package_name or not action_type:
+        try:
+            # TODO(jacobperron): This logic should come from a rosidl related package
+            parts = action_type.split('/')
+            if len(parts) < 2:
```

Comment:
I would suggest to use `== 1` as in all other occurrences in #259.

Reply:
Done 17cdf4e